src/Tribe/Importer/File_Importer.php

```diff
@@ -17,6 +17,7 @@
 	private $updated = 0;
 	private $created = 0;
 	private $skipped = array();
+	private $encoding = array();
```

Comment:
If the `encoding` field is only used as a counter and nothing more then switching from an array to an int would make sense.